package.json

```
@@ -2,7 +2,7 @@
   "name": "40-2nd-devu-backend",
   "version": "1.0.0",
   "description": "",
-  "main": "app.js",
+  "main": "server.js",
```

main은 app.js로 수정해주세요!

```suggestion
  "main": "app.js",
```